### PR TITLE
LRDOCS-7167 Fix code formatting issues

### PR DESCRIPTION
**Pull request by mwilliams2014**
@sez11a here are the code formatting changes. I replaced the JSP language with Markup in a separate commit. The JSP language isn't providing any syntax highlighting on the site. If you don't want to go that route, we can just revert that commit. Thx!

### CODE REVIEW

**Comment by sez11a** [comment]
This is merge conflicted.

**Comment by mwilliams2014** [comment]
@sez11a I rebased and fixed the conflict, but now I'm seeing the same commits listed twice...

File: deployment/articles/04-securing-liferay/05-ldap/01-ldap-servers-intro.markdown [hunk 1]
```diff
@@ -135,12 +135,12 @@ method you use. For example, if you changed @product@'s authentication method to
 use screen names instead of the email addresses, you would modify the search
 filter so it can match the entered log in name:
 
-        (cn=@screen_name@)
+    (cn=@screen_name@)
```
Comment (sez11a) [inline deployment/articles/04-securing-liferay/05-ldap/01-ldap-servers-intro.markdown]: @mwilliams2014 Something's still not right with this.These LDAP queries are code; they should be formatted as such (these are docs I wrote).

File: deployment/articles/04-securing-liferay/05-ldap/02-configuring-ldap.markdown [hunk 2]
```diff
@@ -25,7 +25,7 @@ a user export every time the user logs in. You can prevent updates to users'
 `lastLoginDate` fields from triggering LDAP user exports by setting the
 following property in your `portal-ext.properties` file:
 
-        users.update.last.login=false
+    users.update.last.login=false
```
Comment (sez11a) [inline deployment/articles/04-securing-liferay/05-ldap/02-configuring-ldap.markdown]: @mwilliams2014 Here's a property that's now not code. Did you use some automated tool to do this?

**Comment by mwilliams2014** [comment]
hmmm...I manually updated it, but I had to merge master, I must have reverted some things...let me take another stab at this...

File: deployment/articles/04-securing-liferay/05-ldap/01-ldap-servers-intro.markdown (hunk 1 above)
Comment (mwilliams2014) [reply deployment/articles/04-securing-liferay/05-ldap/01-ldap-servers-intro.markdown]: @sez11a it's still formatted as code. I believe this one was just indented one tab too many

Comment (mwilliams2014) [reply deployment/articles/04-securing-liferay/05-ldap/01-ldap-servers-intro.markdown]: I removed the extra space in the code-block

Comment (mwilliams2014) [reply deployment/articles/04-securing-liferay/05-ldap/01-ldap-servers-intro.markdown]: @sez11a see the extra space here: https://portal.liferay.dev/docs/7-2/deploy/-/knowledge_base/d/ldap#users

File: deployment/articles/04-securing-liferay/05-ldap/02-configuring-ldap.markdown (hunk 2 above)
Comment (mwilliams2014) [reply deployment/articles/04-securing-liferay/05-ldap/02-configuring-ldap.markdown]: @sez11a this is still a code block. same as above. I removed the extra space that wasn't needed

**Comment by mwilliams2014** [comment]
@sez11a this should be good. I removed extra space that wasn't needed

**Comment by mwilliams2014** [comment]
I see several files are conflicted now...I know I put in time on this, but we may just have to scrap it...

**Comment by sez11a** [comment]
This thing is just so big it makes me nervous. Let me play with it locally.

**Comment by mwilliams2014** [comment]
I understand :-) Let me know if you run into any other issues, but it should be good

**Comment by sez11a** [comment]
Merged. Nice work!

**Comment by mwilliams2014** [comment]
Ah, great! thx!

**Pull request merged**